src/components/PageTitle.js

```diff
@@ -1,7 +1,18 @@
 import React from "react";
+import Image from "next/image";
+import starBackground from "@/public/assets/starBackground.svg";
```

Comment:
also add the earth in the background. Check Figma. I added the earth asset in the public folder.